packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -83,7 +83,8 @@ const INTRINSIC_TX_GAS_COST = 21000
 // Additional intrinsic gas for a transaction with fee currency specified
 const ADDITIONAL_INTRINSIC_TX_GAS_COST = 50000
 
-const stableTokenTransferGasCost = 20653
+// const stableTokenTransferGasCost = 20653
```

Comment:
Why is this different?

Reply:
This sometimes changes, haven't found the reason yet...

Reply:
+1 to this. We need to make sure we know why gas costs change.

Reply:
It was because a public method was added to UsingPrecompiles, so the routing code changed.